README.md

```diff
@@ -13,16 +13,13 @@ Supported browsers:
  - Chrome Android
  - Firefox desktop
 
-Unspported browsers:
- - Firefox Android
- - Safari
- - Chrome iOS
+Please note that Firefox Android, Safari, and Chrome on iOS are all unsupported.
```

Comment:
this is good though you could just say "all ios browsers". Also we know FF android has a bug but ios is untested, so we could say unsupported / untested.